packages/syft/tests/syft/core/tensor/smpc/smpc_tensor_test.py

```diff
@@ -0,0 +1,53 @@
+# stdlib
```

Comment:
This is not working!! It works only for Hagrid this version

Reply:
Acceptable